data/converters/converters.go

```diff
@@ -0,0 +1,195 @@
+package converters
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+func toConversionError(expected string, v interface{}) error {
+	return fmt.Errorf("expected %s input but got type %T for value \"%v\"", expected, v, v)
+}
+
+// Int64NOOP The input and output values are both int64.
+var Int64NOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeInt64,
+}
+
+// BoolNOOP The input and output values are both bool.
+var BoolNOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeBool,
+}
+
+// Float64NOOP The input and output values are both int64.
+var Float64NOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeFloat64,
+}
+
+// StringNOOP The input and output values are both string.
+var StringNOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeString,
+}
+
+// AnyToNullableString Converts any non-null value into a string.
+var AnyToNullableString = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableString,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
```

Comment:
Instead it is better to start with `var str *string`.

Then instead of returning nil, return str instead if `v == nil`.

This is because we are returning an interface{}, but a nil interface is not the same as a nil interface that holds a typed nil.

In practice we handle this in the Set method, but better to follow this pattern and be more explicit in the type.

So follow this pattern for the rest (this also fits with updated suggested comment above).